modules/library/referencing/src/main/java/org/geotools/referencing/factory/gridshift/DataUtilities.java

```diff
@@ -35,15 +35,33 @@
  */
 public class DataUtilities {
 
+    private static final boolean IS_WINDOWS_OS = System.getProperty("os.name").toUpperCase()
+            .contains("WINDOWS");
+
     /**
-     * Copy of DataUtilities.urlToFile(URL url) in gt-main, which cannot be used here because of
-     * cyclic Maven dependencies.
+     * Takes a URL and converts it to a File. The attempts to deal with Windows UNC format specific
```

Comment:
Can we make a Files utility method then for this stuff, and then deprecate the method in the DataUtilities class? Having two classes with the same name in different packages is ... confusing.